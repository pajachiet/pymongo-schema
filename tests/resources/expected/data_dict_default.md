
### Database: test_db1
#### Collection: test_col1 
|Field_compact_name     |Field_name        |Full_name     |Description     |Count     |Percentage     |Types_count                                   |
|-----------------------|------------------|--------------|----------------|----------|---------------|----------------------------------------------|
|_id                    |_id               |None          |None            |25359     |100.0          |oid : 25359                                   |
|address                |address           |None          |None            |25359     |100.0          |OBJECT : 25359                                |
| . building            |building          |None          |None            |25359     |100.0          |string : 25359                                |
| . coord               |coord             |None          |None            |25359     |100.0          |ARRAY(float : 50714, null : 2) : 25359        |
| . street              |street            |None          |None            |25359     |100.0          |string : 25359                                |
| . zipcode             |zipcode           |None          |None            |25359     |100.0          |string : 25359                                |
|borough                |borough           |None          |None            |25359     |100.0          |string : 25359                                |
|cuisine                |cuisine           |None          |None            |25359     |100.0          |string : 25359                                |
|grades                 |grades            |None          |None            |25359     |100.0          |ARRAY(OBJECT : 93463, null : 738) : 25359     |
| : date                |date              |None          |None            |93463     |368.56         |date : 93463                                  |
| : grade               |grade             |None          |None            |93463     |368.56         |string : 93463                                |
| : score               |score             |None          |None            |93463     |368.56         |integer : 93450, null : 13                    |
|name                   |name              |None          |None            |25359     |100.0          |string : 25359                                |
|restaurant_id          |restaurant_id     |None          |None            |25359     |100.0          |string : 25359                                |

#### Collection: test_col2 
|Field_compact_name     |Field_name        |Full_name     |Description     |Count     |Percentage     |Types_count                                   |
|-----------------------|------------------|--------------|----------------|----------|---------------|----------------------------------------------|
|_id                    |_id               |None          |None            |25359     |100.0          |oid : 25359                                   |
|address                |address           |None          |None            |25359     |100.0          |OBJECT : 25359                                |
| . building            |building          |None          |None            |25359     |100.0          |string : 25359                                |
| . coord               |coord             |None          |None            |25359     |100.0          |ARRAY(float : 50714, null : 2) : 25359        |
| . street              |street            |None          |None            |25359     |100.0          |string : 25359                                |
| . zipcode             |zipcode           |None          |None            |25359     |100.0          |string : 25359                                |
|borough                |borough           |None          |None            |25359     |100.0          |string : 25359                                |
|cuisine                |cuisine           |None          |None            |25359     |100.0          |string : 25359                                |
|grades                 |grades            |None          |None            |25359     |100.0          |ARRAY(OBJECT : 93463, null : 738) : 25359     |
| : date                |date              |None          |None            |93463     |368.56         |date : 93463                                  |
| : grade               |grade             |None          |None            |93463     |368.56         |string : 93463                                |
| : score               |score             |None          |None            |93463     |368.56         |integer : 93450, null : 13                    |
|name                   |name              |None          |None            |25359     |100.0          |string : 25359                                |
|restaurant_id          |restaurant_id     |None          |None            |25359     |100.0          |string : 25359                                |


### Database: test_db2
#### Collection: test_col 
|Field_compact_name     |Field_name        |Full_name     |Description     |Count     |Percentage     |Types_count                                   |
|-----------------------|------------------|--------------|----------------|----------|---------------|----------------------------------------------|
|_id                    |_id               |None          |None            |25359     |100.0          |oid : 25359                                   |
|address                |address           |None          |None            |25359     |100.0          |OBJECT : 25359                                |
| . building            |building          |None          |None            |25359     |100.0          |string : 25359                                |
| . coord               |coord             |None          |None            |25359     |100.0          |ARRAY(float : 50714, null : 2) : 25359        |
| . street              |street            |None          |None            |25359     |100.0          |string : 25359                                |
| . zipcode             |zipcode           |None          |None            |25359     |100.0          |string : 25359                                |
|borough                |borough           |None          |None            |25359     |100.0          |string : 25359                                |
|cuisine                |cuisine           |None          |None            |25359     |100.0          |string : 25359                                |
|grades                 |grades            |None          |None            |25359     |100.0          |ARRAY(OBJECT : 93463, null : 738) : 25359     |
| : date                |date              |None          |None            |93463     |368.56         |date : 93463                                  |
| : grade               |grade             |None          |None            |93463     |368.56         |string : 93463                                |
| : score               |score             |None          |None            |93463     |368.56         |integer : 93450, null : 13                    |
|name                   |name              |None          |None            |25359     |100.0          |string : 25359                                |
|restaurant_id          |restaurant_id     |None          |None            |25359     |100.0          |string : 25359                                |

