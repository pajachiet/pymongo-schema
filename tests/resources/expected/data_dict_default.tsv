Database	Collection	Field_full_name	Depth	Field_name	Type
test_db1	test_col1	_id	0	_id	oid
test_db1	test_col1	address	0	address	OBJECT
test_db1	test_col1	address.building	1	building	string
test_db1	test_col1	address.coord	1	coord	ARRAY(float)
test_db1	test_col1	address.street	1	street	string
test_db1	test_col1	address.zipcode	1	zipcode	string
test_db1	test_col1	borough	0	borough	string
test_db1	test_col1	cuisine	0	cuisine	string
test_db1	test_col1	grades	0	grades	ARRAY(OBJECT)
test_db1	test_col1	grades:date	1	date	date
test_db1	test_col1	grades:grade	1	grade	string
test_db1	test_col1	grades:score	1	score	integer
test_db1	test_col1	name	0	name	string
test_db1	test_col1	restaurant_id	0	restaurant_id	string
test_db1	test_col2	_id	0	_id	oid
test_db1	test_col2	address	0	address	OBJECT
test_db1	test_col2	address.building	1	building	string
test_db1	test_col2	address.coord	1	coord	ARRAY(float)
test_db1	test_col2	address.street	1	street	string
test_db1	test_col2	address.zipcode	1	zipcode	string
test_db1	test_col2	borough	0	borough	string
test_db1	test_col2	cuisine	0	cuisine	string
test_db1	test_col2	grades	0	grades	ARRAY(OBJECT)
test_db1	test_col2	grades:date	1	date	date
test_db1	test_col2	grades:grade	1	grade	string
test_db1	test_col2	grades:score	1	score	integer
test_db1	test_col2	name	0	name	string
test_db1	test_col2	restaurant_id	0	restaurant_id	string
test_db2	test_col	_id	0	_id	oid
test_db2	test_col	address	0	address	OBJECT
test_db2	test_col	address.building	1	building	string
test_db2	test_col	address.coord	1	coord	ARRAY(float)
test_db2	test_col	address.street	1	street	string
test_db2	test_col	address.zipcode	1	zipcode	string
test_db2	test_col	borough	0	borough	string
test_db2	test_col	cuisine	0	cuisine	string
test_db2	test_col	grades	0	grades	ARRAY(OBJECT)
test_db2	test_col	grades:date	1	date	date
test_db2	test_col	grades:grade	1	grade	string
test_db2	test_col	grades:score	1	score	integer
test_db2	test_col	name	0	name	string
test_db2	test_col	restaurant_id	0	restaurant_id	string
